### PR TITLE
Eng 1869 system level bundles are installed without adding components inside the bundle own folder

### PR DESCRIPTION
**Pull request by firegloves**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **firegloves**

If they are not already assigned, you can assign the PR to them by writing `/assign @firegloves` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 5ac707428a66fa462018752f3f71378c4b88378a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | b5e918ac43aea94d7a2c7f1b432177548ddbbf1a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@firegloves: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | ffccbc6f775f81478c3d22c7c92d552dd26c5c7d | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr103**

File: src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java:120]: I don't get the rename from **resourceFolder** to **resourceDir**

you wanted to avoid confusion with resourceFolders?

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java:283]: probably be better to assign an explicit value to the enum

File: src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java
Comment (firegloves) [reply src/main/java/org/entando/kubernetes/model/bundle/processor/DirectoryProcessor.java:120]: Because I'm using resouceFolder as var name in the step at line 113, that in my opinion is more important that has a meaningful name rather than the one in a for loop

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-component-manager-ecmpr103**

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java
Comment (firegloves) [reply src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java]: updated

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-component-manager/pull/103#pullrequestreview-602990709" title="Approved">w-caffiero-entando</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)~~ [w-caffiero-entando]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
